src/linux/process.rs

```diff
@@ -431,6 +419,7 @@ pub(crate) fn _get_process_data(
     refresh_kind: ProcessRefreshKind,
 ) -> Result<(Option<Process>, Pid), ()> {
     let pid = match path.file_name().and_then(|x| x.to_str()).map(Pid::from_str) {
+        // If `pid` and `nb` are the same, it means the file is linking to itself so we skip it.
```

Comment:
Could you add some clarification here? Is this related to the "Idle state" changes, or something else? Is there a source for this behavior? (...is it particular to kernel threads?)

Reply:
Oh I can but it's not directly linked to the PR, just that when reading it, I realized that a comment would be nice.

So in this case, we are reading a process sub-folders to look for children. If a children folder's number is the same as the parent, then no need to go into it since it's a link to itself.